plugins/polio/js/src/domains/Chronogram/Chronogram/index.tsx

```diff
@@ -22,16 +24,27 @@ export const Chronogram: FunctionComponent = () => {
     const classes: Record<string, string> = useStyles();
     const { formatMessage } = useSafeIntl();
 
+    const { data: chronogramMetaData, isFetching: isFetchingMetaData } =
+        useOptionChronogram();
+
```

Comment:
You could call that hook directly in `ChronogramFilters` since you don't use the data here and just pass it as props